apps/01PigLatin.js

```diff
@@ -7,11 +7,92 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: You want to set the sting to all lower case so that if a user uses CAPS then it doens't
+  // matter becasue it will revert to lower case and the program will not crash.
```

Comment:
It's not about crashing the program.  It's about reducing the number of vowels we have to check. `'a' !== 'A'`.

Close though.

![duncan](http://i.imgur.com/9WqlsIL.gif)